AmazonChimeSDK/AmazonChimeSDK/internal/contentshare/DefaultContentShareVideoClientController.swift

```diff
@@ -43,6 +43,12 @@ import Foundation
     }
 
     public func startVideoShare(source: VideoSource) {
+        logger.info(msg: "Starting video share with video source")
+        startVideoShare(source: source, config: LocalVideoConfiguration())
+    }
+
+    public func startVideoShare(source: VideoSource, config: LocalVideoConfiguration) {
+        // ignore LocalVideoConfiguration because contentshare does not have simulcast
```

Comment:
nit: I think it's better to move this comment to `startContentShare()` in `ContentShareController.swift` as documentation comment, maybe also add it to `LocalVideoConfiguration.simulcastEnabled`.